sources/packages/backend/apps/api/src/services/education-program-offering/custom-validators/has-valid-offering-period-for-funded-days.ts

```diff
@@ -23,20 +20,32 @@ class HasValidOfferingPeriodForFundedDaysConstraint
   implements ValidatorConstraintInterface
 {
   validate(studyBreaks: StudyBreak[], args: ValidationArguments): boolean {
+    const offeringMinDaysAllowedValue = this.getOfferingMinAllowedDays(args);
```

Comment:
Nice implementation to pass the args from the form with respect to the offering intensity, but can this be shared done as a constant without passing them from the form?